automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/workspace/WorkspaceSpec.scala

```diff
@@ -335,43 +335,47 @@ class WorkspaceSpec extends FreeSpec with WebBrowserSpec with WorkspaceFixtures
 
     "who has writer access" - {
       "and does not have canCompute permission" - {
-        "should see launch analysis button disabled" ignore withWebDriver { implicit driver =>
-          val Seq(user1, user2) = UserPool.chooseStudents(2)
-          implicit val authToken: AuthToken = Config.Users.owner.makeAuthToken()
-          withWorkspace(billingProject, "WorkspaceSpect_writerAccess") { workspaceName =>
-            withSignIn(user1) { listPage =>
-              api.methodConfigurations.createMethodConfigInWorkspace(billingProject, workspaceName, SimpleMethod, SimpleMethodConfig.configNamespace, s"$methodConfigName", 1,
+        "should see launch analysis button disabled" in withWebDriver { implicit driver =>
+          val user = UserPool.chooseStudent
+          val owner = UserPool.chooseProjectOwner
+          implicit val authToken: AuthToken = authTokenOwner
```

Comment:
If I'm not mistaken, this looks like it's choosing a projectOwner but then using the auth token from the projectOwner at the class level.

Reply:
nice catch!